xbmc/utils/LangCodeExpander.cpp

```diff
@@ -234,6 +242,78 @@ bool CLangCodeExpander::ConvertToISO6392T(const std::string& strCharCode,
   return false;
 }
 
+//Lookup ISO-3166-1 table and return codes or name.
+bool CLangCodeExpander::LookupISO31661(const std::string& saught,
+                                       std::string& returned,
+                                       bool returnCode)
+{
+  returned = "";
+
+  if (saught.size() == 2) //Lookup based on ISO-3166-1 Alpha 2
+  {
+    for (const auto& codes : RegionNames)
```

Comment:
use `auto` only for complex namespaces or iterators, avoid use it for simple use cases like this

this and all others cases below

Reply:
auto was used for consistency with other functions in the module.

Reply:
your reply has no sense,
our code guidelines changes over the time
so you must read current code guideline and address this appropriately

Reply:
It makes perfect sense, it explains why I did it that way.
This has been addressed and will be reflected in my next PR update.